api/src/pcapi/routes/backoffice_v3/offers/blueprint.py

```diff
@@ -37,8 +37,69 @@
 )
 
 
-def _get_offers(form: offer_forms.GetOffersListForm) -> list[offers_models.Offer]:
-    base_query = offers_models.Offer.query.options(
+SEARCH_FIELD_TO_PYTHON = {
+    "CATEGORY": {
+        "field": "category",
+        "column": offers_models.Offer.subcategoryId,
+        "special": lambda l: [ subcategory.id for subcategory in subcategories_v2.ALL_SUBCATEGORIES if subcategory.category.id in l],
+    },
+    "CREATION": {
```

Comment:
```suggestion
    "CREATION_DATE": {
```
C'est pas mieux de préciser que c'est une date ?

Reply:
c'est précisé dans la structure. Mais comme on ne précise jamais le type, je ne l'ai pas mis ici non plus.

Reply:
Je trouve ça plus lisible dans le reste du code où on utilise "CREATION", mais soit